OWML.Launcher/App.cs

```diff
@@ -37,7 +37,6 @@ public class App
         public void Run(string[] args)
         {
             _writer.WriteLine($"Started OWML v{_owmlManifest.Version}");
-            _writer.WriteLine("For detailed log, see Logs/OWML.Log.txt");
```

Comment:
you could create the filename already, if you wanted to keep this. your call.

Reply:
this is in the app side, you you'd have to somehow get that path to the unity side... i think it's simpler to create it unity side